website/docs/cloud-docs/workspaces/state.mdx

```diff
@@ -25,6 +25,152 @@ In addition to the current state, Terraform Cloud retains historical state versi
 
 You can view a workspace's state versions from its **States** tab. Each state in the list indicates which run and which VCS commit (if applicable) it was associated with. Click a state in the list for more details, including a diff against the previous state and a link to the raw state file.
 
+## Managed Resources Count
+
+-> **Note:** A managed resources count for each organization is available in the [Terraform Cloud Business tier](https://cloud.hashicorp.com/products/terraform/pricing).
+
+Your organization’s managed resource count helps you understand the number of infrastructure resources that Terraform Cloud manages across all your workspaces.
+
+Terraform Cloud reads all the workspaces’ state files to determine the total number of managed resources. Each [resource](/terraform/language/resources/syntax) instance in the state equals one managed resource. Terraform Cloud does not include [data sources](/terraform/language/data-sources) in the count.
+
+### Examples - Managed Resources
+
+The following Terraform state excerpt describes a  `random` resource. Terraform Cloud counts  `random` as one managed resource because `“mode”: “managed”`.
```

Comment:
```suggestion
The following Terraform state excerpt describes a  `random` resource. Terraform Cloud counts  `random` as one managed resource because `“mode”: “managed”`.

```